subprojects/enterprise-operations/src/main/java/org/gradle/operations/execution/ExecuteWorkBuildOperationType.java

```diff
@@ -19,6 +19,7 @@
 import org.gradle.internal.operations.BuildOperationType;
 
 import javax.annotation.Nullable;
+import java.util.List;
```

Comment:
:x: I think we should move this class also to `org.gradle.enterprise.operations.execution` along with anything we touch that is not already consumed by the GE plugin.